### PR TITLE
Fullcovmat

### PR DESCRIPTION
**Pull request by clementhelsens**
BEGINRELEASENOTES
This PR is including the full covariance from delphes.
As of today it only works on delphes master, thus the failing checks

ENDRELEASENOTES

### CODE REVIEW

**Comment by clementhelsens** [comment]
So that we keep track of this, from delphes 3.4.3pre09 the full track covariance information is stored. This is needed for this PR to succeed tests. But once approved, we are not compatible any-longer with earlier versions. Some more complex mechanism still needs to be deployed

File: converter/src/DelphesEDM4HepConverter.cc [hunk 1]
```diff
@@ -481,16 +482,39 @@ edm4hep::Track convertTrack(Track const* cand, const double magFieldBz)
...
+
+  double scale0 = 1.e3;
+  double scale1 = 1.;
+  //double scale2 = 1.e-3;
+  double scale2 = -2.* 1.e-3;    // CAREFUL: DELPHES USES THE HALF-CURVATURE
+  double scale3 = 1.e3;
+  double scale4 = 1.;
+
+  covMatrix[0]  = covaFB(0,0) *scale0 * scale0;
+  covMatrix[1]  = covaFB(0,1) *scale0 * scale1;
```
Comment (andresailer) [inline converter/src/DelphesEDM4HepConverter.cc:500]: Please don't forget https://github.com/key4hep/EDM4hep/pull/108, I.e, this matrix needs to be transposed

Comment (clementhelsens) [reply converter/src/DelphesEDM4HepConverter.cc:500]: thanks @andresailer , seems I forgot indeed, will transpose as soon as possible

Comment (clementhelsens) [reply converter/src/DelphesEDM4HepConverter.cc]: just did!

**Comment by clementhelsens** [comment]
Remove WIP to prepare for tag

**Pull request merged**